deploy/web/index.html

```diff
@@ -122,6 +122,7 @@ <h1>Bevy Explorer</h1>
         </div>
     <canvas id="mygame-canvas" width="1280" height="720"></canvas> 
     
+    <script src="https://cdn.jsdelivr.net/npm/livekit-client/dist/livekit-client.umd.min.js"></script>

```

Comment:
@robtfm please check this
this is how I'm adding LiveKit JS SDK to the project. I could make it fetch on runtime, and only if I'm with the livekit feature + wasm...

Let me know what you think.

Reply:
as discussed, let's vendor it locally. don't worry about runtime fetching, it's only 100k which is like 0.2% of the wasm and i don't think we will ever want the feature disabled in releases anyway.